File: rxjava/src/main/java/com/asemlab/samples/rxjava/models/Flags.java
package com.asemlab.samples.rxjava.models;

import androidx.annotation.Keep;
import com.google.gson.annotations.SerializedName;

@Keep
public class Flags{

    @SerializedName("alt")
    public String alt;

    @SerializedName("png")
    public String png;

    @SerializedName("svg")
    public String svg;

}

File: chucker/src/main/java/com/asemlab/samples/chucker/models/Flags.kt
package com.asemlab.samples.chucker.models


import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

@Keep
data class Flags(
    @SerializedName("alt")
    val alt: String? = null,
    @SerializedName("png")
    val png: String? = null,
    @SerializedName("svg")
    val svg: String? = null
)